src/components/Footer/socialMediaLinks/SocialMediaLinks.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { SocialMediaIconsReact } from "social-media-icons-react";
+
+function SocialMediaLinks({ icon, backgroundColor }) {
+  const links = [
```

Comment:
This will get created on every render, This is simply an ENUM so you can move it out of the componetn.
Also, name constants in uppercase `LINKS`

Reply:
Ok Jatin.